.github/workflows/linters.yml

```diff
@@ -0,0 +1,29 @@
+name: Linters
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
Maybe give this a name like all the other actions (something like github checkout)